terraform/modules/bulk_upload/access_log_bucket.tf

```diff
@@ -0,0 +1,101 @@
+#tfsec:ignore:aws-s3-enable-bucket-encryption: default bucket encryption is sufficient

```

Comment:
Do we want to update the explanations for this (and also in line 3) because KMS is actually not supported for access logging (as commented [here](https://dluhcdigital.atlassian.net/jira/software/c/projects/CLDC/boards/115?selectedIssue=CLDC-2917&sprints=619))?

Reply:
Nice spot, yes I think we should so have updated it now 😊